portfolio/src/utils/constants.js

```diff
@@ -1,6 +1,6 @@
 
-const WEBSITE_URL = "https://8080-b83fc153-d2cf-481d-a321-9342cdf80f21.us-east1.cloudshell.dev"
-// const WEBSITE_URL = "http://capparelli-step-2020.appspot.com"
+//const WEBSITE_URL = "https://8080-b83fc153-d2cf-481d-a321-9342cdf80f21.us-east1.cloudshell.dev"
```

Comment:
remove test code?

Reply:
I have these there because I will get a CORS error if I try to reach appspot during testing, though I guess I could keep the test one in a personal file or something